tests/testthat/test-renderer1-tooltip.R

```diff
@@ -107,22 +108,13 @@ info <- animint2HTML(viz)
 
 test_that("tooltip works with href",{
   # Test for bug when points are not rendered with both href + tooltip
-  
-  # See that points are rendered for every country. This test may need to be
-  # changed soon!!! See the viz on the link below:
-  # http://bl.ocks.org/faizan-khan-iit/449b84b2e6b55ca5e39ea9aaad97e27e
-  # You may notice a point on the very top-left of the viz, outside the plot.
-  # These are the points that have NaN as their value in the DOM, arriving from
-  # NA values in the tsv. Should be an easy fix. But leaving a comment here
```

Comment:
@tdhock As I was testing the axis updates, [with this WorldBank example](http://bl.ocks.org/faizan-khan-iit/raw/a3e19ad00691d2dafffca97e9e4b0576/), I actually got seemingly wrong range values for zooming of geoms. E.g. upon selecting North America, the geoms do not cover the whole PANEL area... Removing NAs from the original data seemed to work fine.
Basically we remove the NAs before saving to tsv now, is that correct? In that case this example should also work fine..